### ISSUE TITLE
Invalid rendering of form-append/prepend icons

### ISSUE THREAD

**Issue by syberon** [neilime/zf-twbs-helper-module#23]
I found an error of rendering form-append/prepend icons in version 2.0.

This happens because function (FormButton.php, line 351):
`$sIconContent = $this->htmlElement('i', ['class' => $aIconOptions['class']]);`
returns unclosed `<i>` tag and it brokes all markup.

Trait HtmlTrait.php returns unclosed tag because it has empty content:
```
$sElementContent = '<' . $sTag . $sAttributes;
if ($sContent === null) {
    $sElementContent .= $this->getView()->plugin('HtmlTag')->getClosingBracket();
    return $sElementContent;
}
```

**Comment by neilime** [comment]
Fixed in version v2.0.1

**Comment by syberon** [comment]
Thanks for fast response :) 
I didn’t make my PR, because I see that you started to actively develop this package again.
